src/data/petsData.ts

```diff
@@ -74,6 +77,7 @@ export const petsData: PetProps[] = [
   },
```

Comment:
there is wrong image numeration in in public/images/fromLove/forest, we need to fix it!